andela/bp-esa-frontend
# #165998462 Add dropdown for statistics

#### What does this PR do?
- This PR adds a dropdown in the statistics section to enable different time periods to be selected for statistics to be carried out.

#### Description of Task to be completed?
- Add a dropdown with different time periods
- Update the statistics route to include a variable in which a time period is passed to the backend.

#### How should this be manually tested?
1. Git clone the repository.
2. Checkout to the `ft-statistics-dropdown-165998462` branch.
5. Start the server `npm run start` _(or the `yarn` equivalent)_.
6. Clone the backend repository, `https://github.com/andela/bp-esa-backend`.
7. Checkout to the backend `ft-add-statistics-period-165998462` branch.
8. Start the backend server `npm run start:dev` _(or the `yarn` equivalent)_.
9. Login to the ESA application
10. Click the caret in the statistics section.
11. Click a time period.
The statistics for the selected time period should be displayed.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[165998462](https://www.pivotaltracker.com/story/show/165998462)

#### Screenshots (If applicable)
<img width="1403" alt="Screen Shot 2019-05-16 at 15 44 56" src="https://user-images.githubusercontent.com/38077368/57854747-a3c60c80-77f1-11e9-9846-3fcfdc198dfd.png">
